addons/web/static/src/core/user.js

```diff
@@ -197,7 +197,11 @@ export function _makeUser(session) {
             return Object.assign({}, context, { uid: this.userId });
         },
         get evalContext() {
-            return { uid: this.userId, companies: companyEvalContext };
+            return {
+                uid: this.userId,
+                companies: companyEvalContext,
+                allowed_company_ids: this.context.allowed_company_ids, //Deprecated, to be removed !!!
```

Comment:
I bet ``allowed_company_ids`` would have been better to keep (or ``allowed_companies``) as it has less conflicts when grepping. Companies is such a generic name you can't grep it in code...

Reply:
grep: `companies.active_id`